src/app/(main)/[slug]/page.tsx

```diff
@@ -53,15 +62,27 @@ async function Page({ params }: Props) {
     initialBlogPage,
     initialCompensationsPage,
     initialLocationsData,
+    initialCustomerCases,
+    initialSharedCustomerCases,
   ] = await Promise.all([
-    loadQuery<PageBuilder>(SLUG_QUERY, { slug }, { perspective }),
-    loadQuery<BlogPage>(BLOG_PAGE_QUERY, { slug }, { perspective }),
-    loadQuery<CompensationsPage>(
+    loadStudioQuery<PageBuilder>(SLUG_QUERY, { slug }, { perspective }),
+    loadStudioQuery<BlogPage>(BLOG_PAGE_QUERY, { slug }, { perspective }),
+    loadStudioQuery<CompensationsPage>(
       COMPENSATIONS_PAGE_QUERY,
       { slug },
       { perspective },
     ),
-    loadQuery<CompanyLocation[]>(COMPANY_LOCATIONS_QUERY, {}, { perspective }),
+    loadStudioQuery<CompanyLocation[]>(
+      COMPANY_LOCATIONS_QUERY,
+      {},
+      { perspective },
+    ),
+    loadStudioQuery<CustomerCasePage>(
+      CUSTOMER_CASES_PAGE_QUERY,
+      { slug },
+      { perspective },
+    ),
+    loadSharedQuery<CustomerCase[]>(CUSTOMER_CASES_QUERY, {}, { perspective }),
```

Comment:
I think this a better approach would be to fetch inside the costumer cases page, I did it poorly with blog posts.